app/controllers/answers_controller.rb

```diff
@@ -39,6 +66,10 @@ def load_question
   end
 
   def answer_params
-    params.require(:answer).permit(:body, :user_id, attachments_attributes: [:file])
+    params.require(:answer).permit(:body, :user_id, attachments_attributes: [:file, :_destroy])
+  end
+
+  def load_favorite
+    @favorite_answer = @question.favorite_answer.nil? ? nil : @question.answers.find(@question.favorite_answer)
```

Comment:
Maybe it would be shorter:
`@favorite_answer = @question.answers.find_by(id: @question.favorite_answer&.id)`

Reply:
Yes, thanks)